README.md

```diff
@@ -240,6 +265,7 @@ If you want `coauthor@yourhostname.com` to receive email,
 add an alias like `coauthor: edemaine@mit.edu` to `/etc/aliases`
 and then run `sudo newaliases`.
 
+## Performance monitoring ###
```

Comment:
Of course this needs to be updated at some point (https://github.com/edemaine/coauthor/issues/204), just changing the style for the moment.